src/main/java/software/amazon/cloudwatchlogs/emf/model/MetricDirective.java

```diff
@@ -67,14 +67,22 @@ void putDimensionSet(DimensionSet dimensionSet) {
     }
 
     void putMetric(String key, double value) {
-        putMetric(key, value, Unit.NONE);
+        putMetric(key, value, Unit.NONE, StorageResolution.STANDARD);
     }
 
     void putMetric(String key, double value, Unit unit) {
```

Comment:
Looks like this method is never used (except in test package)

Reply:
I understand your concern. However, I added this method just to maintain consistency with the existing code base.

There are some existing method definitions in this class as below which are never used too:
1. putMetric(String key, double value)
2. putMetric(String key, double value, Unit unit)

Only one putMetric() [method definition](https://github.com/awslabs/aws-embedded-metrics-java/pull/129/files#diff-c6f7210af6ccdb7ae92d7482d34737004b81d350937bd8b9d68ffceb75efd5a9R81) is used from this class. Let me know if you think we should still remove this unused method definition.

Reply:
These methods are used to help with tests. Added a comment to specify that these are helper methods.